src/main/java/com/everylingo/everylingoapp/service/TranslationRequestService.java

```diff
@@ -0,0 +1,45 @@
+package com.everylingo.everylingoapp.service;
+
+import com.everylingo.everylingoapp.exception.UserAlreadyExistsException;
+import com.everylingo.everylingoapp.model.AppUser;
+import com.everylingo.everylingoapp.model.TranslationRequest;
+import com.everylingo.everylingoapp.repository.TranslationRequestRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TranslationRequestService {
+    @Autowired
+    private TranslationRequestRepository translationRequestRepository;
+
+    public void saveRequest(TranslationRequest translationRequest) {
+        translationRequestRepository.save(translationRequest);
+    }
+
+    public List<TranslationRequest> getAllRequests()
+    {
+        List<TranslationRequest> translationRequests = new ArrayList<TranslationRequest>();
```

Comment:
Seems unnecessary, you could simply return the result provided by the findAll() method